public/app/features/scopes/tests/utils/actions.ts

```diff
@@ -35,7 +35,8 @@ const type = async (selector: () => HTMLInputElement, value: string) => {
   await jest.runOnlyPendingTimersAsync();
 };
 
-export const updateScopes = async (scopes: string[]) => act(async () => ScopesService.instance?.changeScopes(scopes));
+export const updateScopes = async (service: ScopesService, scopes: string[]) =>
```

Comment:
Utils have to get the services now instead of accessing the singleton instances.